packages/jaeger-ui/src/components/TracePage/TracePageHeader.css

```diff
@@ -19,11 +19,26 @@ limitations under the License.
   display: flex;
 }
 
+Divider {
+  color: red;
+}
+
+.TracePageHeader--titleRowEmbed {
+  align-items: center;
+  background-color: #ececec;
+  border-bottom: 1px solid #d8d8d8;
+  display: flex;
+}
+
 .TracePageHeader--title {
   margin: 0;
   padding: 0.25rem 0.5rem;
 }
 
+.TracePageHeader--titleEmbed {
```

Comment:
Please keep related styles adjacent, e.g. `.TracePageHeader--title` and `.TracePageHeader--title:hover`

Reply:
Side note, since these are similar and you don't want the hover, writing it as:

```css
.TracePageHeader--title,
.TracePageHeader--titleEmbed {
  margin: 0;
  padding: 0.25rem 0.5rem;
}
```